packages/plugin-core/src/test/suite-integ/NoteLookupCommand.test.ts

```diff
@@ -794,14 +794,15 @@ suite("NoteLookupCommand", function () {
           });
           const document = VSCodeUtils.getActiveTextEditor()?.document;
           const newNote = WSUtils.getNoteFromDocument(document!);
+          const currentDate = Time.now();
```

Comment:
this test is problematic because it runs the same logic to test as the code. better to stub the date and then test if the format is correct. see https://stackoverflow.com/questions/31591098/how-do-i-stub-new-date-using-sinon

Reply:
fixed